### PR TITLE
fix: profile loading fix in org

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes https://threads.com/34539679188?s=po6sYHCBaUoV1MzZqh55t1

Screenshot 2023-09-29 at 9 50 49 PM

E2E Test:-

Screenshot 2023-09-29 at 11 08 18 PM

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/pages/settings/my-account/profile.tsx [hunk 1]
```diff
@@ -90,9 +90,13 @@ const ProfileView = () => {
...
-      fetch(userData.avatar).then((res) => {
-        if (res.url) setFetchedImgSrc(res.url);
-      });
+      if (!userData.organization) {
+        fetch(userData.avatar).then((res) => {
+          if (res.url) setFetchedImgSrc(res.url);
+        });
+      } else {
+        setFetchedImgSrc("ORG_PROFILE_IMAGE");
+      }
```
Comment (Udit-takkar) [inline apps/web/pages/settings/my-account/profile.tsx:99]: don't need this for org

Comment (Udit-takkar) [reply apps/web/pages/settings/my-account/profile.tsx:99]: fetchedImgSrc was used to check if user has any set any image or not. url ending with /avatar.svg is allback image. (earlier it was gravatar url )

Comment (leog) [reply apps/web/pages/settings/my-account/profile.tsx:99]: NIT: feels weird to set a state to an unused string value, isn't there any other way? 🤔

**Review by leog — APPROVED** [review]
LGTM, left a NIT

**Comment by Udit-takkar** [comment]
@leog i added a any random string to prevent loading state.  fetchedImgSrc is not used anywhere so we can just set it any value

https://github.com/calcom/cal.com/blob/60578a2a33d683dd1996840296fdb713eb729b88/apps/web/pages/settings/my-account/profile.tsx#L225

**Review by leog — CHANGES_REQUESTED** [review]
@Udit-takkar understood about the random string, but it would be much better to be able to assign `undefined` to it.

Also, shifting to test-policeman mode, we need a test for this to comply with our new testing standards 🙏 

image

**Review by leog — APPROVED** [review]
Great job, LGTM

**Comment by PeerRich** [comment]
CleanShot 2023-09-29 at 20 56 47@2x
what is going on here?

**Comment by leog** [comment]
> what is going on here?

@PeerRich the active and hover states of those fields don't match it seems, but not related to this PR though, we could create a bug for it to follow up

**Pull request merged**